modules/client/src/main/ts/api/Bdd.ts

```diff
@@ -0,0 +1,36 @@
+import { Global } from '@ephox/bedrock-common';
+import { MochaGlobals } from 'mocha';
+
+const mochaGlobals: MochaGlobals = Global;
+
+const before = mochaGlobals.before;
```

Comment:
Would a destructuring assignment be nicer here?

Reply:
It would be, however from a tree shaking perspective having them as separate variables would allow the unused variables to be dropped (have a tendency to do this because of that). Anyways, that doesn't matter here as we'd also need to mark it as side-effect free, so I'll go ahead change this to use object destructing instead.

Reply:
Fixed